app/views/assignments/_test_form.html.erb

```diff
@@ -1,67 +1,80 @@
 <%= form_with model: [@assignment, @test], local: true do |form| %>
-    <div class="form-grid">
+  <div class="form-grid">
     <div class="form-group">
-        <%= form.label :test_grouping_position, "Group Number" %>
-        <%= form.text_field :test_grouping_position, class: "form-control" %>
+      <%= form.label :test_grouping_position, "Group Number" %>
+      <%= form.text_field :test_grouping_position, class: "form-control" %>
     </div>
     <div class = "form-group">
-        <%= form.label :name %>
+      <%= form.label :name %>
         <%= form.text_field :name, class:"form-control" %>
     </div>
     <div class = "form-group">
-        <%= form.label :points %>
+      <%= form.label :points %>
         <%= form.number_field :points, class:"form-control" %>
     </div>
     <div class = "form-group">
-        <%= form.label :test_type %>
-        <%= form.select :test_type, options_for_select([['coverage', 'coverage'], ['compile', 'compile'], ['approved_includes', 'approved_includes'], ['memory_errors', 'memory_errors'], ['unit', 'unit'], ['i/o', 'i/o'], ['performance', 'performance'], ['script', 'script']], @test.test_type), class:"form-control" %>
+      <%= form.label :test_type %>
+      <%= form.select :test_type, 
+                      options_for_select([['coverage', 'coverage'], 
+                                          ['compile', 'compile'], 
+                                          ['approved_includes', 'approved_includes'], 
+                                          ['memory_errors', 'memory_errors'], 
+                                 ['unit', 'unit'], 
+                                          ['i_o', 'i_o'], 
```

Comment:
The first element in each of these pairs is the display text. Let's make these less backend-y:
`Coverage`, `Compile`, `Approved Includes`, `Memory Errors`, `Unit`, `I/O`, `Performance`, `Script`

Reply:
This is gonna be a large work because the relevant cucumber tests are based on the selection (Currently the
selection is like this if the display text changes then the type needs to be changed(approved_includes to Approved Includes for example). Can this be put off and make sure other functions work first?
  | type              | name  | points | target      |
        | approved_includes | test1 | 10     | target1.cpp |
        | compile           | test2 | 10     | target2.cpp |
        | memory_errors     | test3 | 10     | target3.cpp |
        | coverage          | test4 | 10     | target4.cpp |
        | unit              | test5 | 10     | target5.cpp |
        | i_o               | test6 | 10     | target6.cpp |
        | performance       | test7 | 10     | target7.cpp |
        | script            | test8 | 10     | target8.cpp |)

Reply:
The left is the display text, keep the right the same, and it should be fine

Reply:
Should be a very easy fix

Reply:
Issue created. This will be done in a separate PR.

Reply:
#139